new package: icecream 2.1.8

totaam opened the pull request:
This is needed by paramiko 4, which will be updated next.

striezel reviewed (approved):
Looks mostly good to me.

However, it is odd that the test command in the `check()` function is commented out. Haven't tested it myself, but the [PKGBUILD for python-icecream in Arch Linux](https://gitlab.archlinux.org/archlinux/packaging/packages/python-icecream/-/blob/main/PKGBUILD?ref_type=heads) currently has this code for `check()`:

```
check() {
  cd "$pkgname"

  python -m unittest
}
```

Maybe we can use the `python -m unittest` part here, too? I'll merge anyway, because check() is not executed in CI at the moment.

totaam commented:
@striezel forgot to mention that the unit tests are commented out because one of them fails - probably assumes unix style paths:
```
tests/test_icecream.py .......F..............................ss          [ 95%]
tests/test_install.py ..                                                 [100%]

================================== FAILURES ===================================
__________________ TestIceCream.testContextAbsPathMultiLine ___________________

self = <tests.test_icecream.TestIceCream testMethod=testContextAbsPathMultiLine>

    def testContextAbsPathMultiLine(self):
        multilineStr = 'line1\nline2'
        with configureIcecreamOutput(includeContext=True, contextAbsPath=True):
            with disableColoring(), captureStandardStreams() as (out, err):
                ic(multilineStr)
    
        firstLine = err.getvalue().splitlines()[0]
>       assert lineIsAbsPathContext(firstLine)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/test_icecream.py:492: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

line = 'E:/MINGW-packages/mingw-w64-python-icecream/src/python-build-MINGW64/tests/test_icecream.py:489 in testContextAbsPathMultiLine()'

    def lineIsAbsPathContext(line):
        line = stripPrefix(line)  # ic| /absolute/path/to/f.py:33 in foo()
        sourceLocation, function = line.split(' in ')  # /absolute/path/to/f.py:33 in foo()
>       filepath, lineNumber = sourceLocation.split(':')  # /absolute/path/to/f.py:33
        ^^^^^^^^^^^^^^^^^^^^
E       ValueError: too many values to unpack (expected 2)

tests/test_icecream.py:136: ValueError
=========================== short test summary info ===========================
FAILED tests/test_icecream.py::TestIceCream::testContextAbsPathMultiLine - Va...
=================== 1 failed, 39 passed, 2 skipped in 1.42s ===================
==> ERROR: A failure occurred in check().
    Aborting...
```
And I don't really care for it since the project I want to build with it (paramiko 4) passes its own unit tests despite this one failure here.
